### PR TITLE
Christmas Nuke Song

### PR DESCRIPTION
**Pull request by Piras314**
## About the PR

Adds nukemass by mrjajkes to the nuke songs for Christmas.

## Media

393427159-de3e209a-1b6b-4d55-92ab-14072517775b

https://www.youtube.com/watch?v=gLSyt7AaqQM

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: mrjajkes, Piras314
- add: Added Nukemass by mrjajkes to the nuke songs for christmas!

### CODE REVIEW

**Comment by he1acdvv** [comment]
image

this btw.

**Comment by Piras314** [comment]
@Geekyhobo :godo:

**Comment by ScarKy0** [comment]
Yes please this is so peak

File: Resources/Audio/StationEvents/attributions.yml [hunk 1]
```diff
@@ -11,4 +11,9 @@
 - files: ["sound_station_14.ogg"]
   license: "CC-BY-3.0"
   copyright: "Created by Donchan"
-  source: "https://www.youtube.com/watch?v=4nUpeYLKUns"
\ No newline at end of file
+  source: "https://www.youtube.com/watch?v=4nUpeYLKUns"
+
+- files: ["nukemass.ogg"]
+  license: "CC-BY-3.0"
+  copyright: "Created by Mrjajkes"
```
Comment (slarticodefast) [inline Resources/Audio/StationEvents/attributions.yml:18]:
```suggestion
  copyright: "Created by mrjajkes"
```
I asked mrjajkes for confirmation that this is the correct licence. But could you fix the name in the attribution like they requested?
grafik

**Review by slarticodefast — CHANGES_REQUESTED** [review]

**Review by slarticodefast — APPROVED** [review]

**Comment by slarticodefast** [comment]
@Geekyhobo Fulfill your promise 😄

**Comment by Geekyhobo** [comment]
Hi hi hi merry Christmas I approve this pr

**Comment by chromiumboy** [comment]
Oh yes, this is awesome

**Pull request merged**